packages/cosmjs/README.md

```diff
@@ -18,10 +18,10 @@ $ npm install @turnkey/cosmjs
 
 ## Examples
 
-| Example                                 | Description                                                                                       |
-| --------------------------------------- | ------------------------------------------------------------------------------------------------- |
-| [`with-cosmjs`](/examples/with-cosmjs/) | Create a new Cosmos address, then sign and broadcast a transaction on Celestia testnet via CosmJS |
+| Example                                      | Description                                                                                       |
+| -------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| [`with-cosmjs`](../../examples/with-cosmjs/) | Create a new Cosmos address, then sign and broadcast a transaction on Celestia testnet via CosmJS |
 
 ## See also
 
-- [`@turnkey/http`](/packages/http/): lower-level fully typed HTTP client for interacting with Turnkey API
+- [`@turnkey/http`](../http/): lower-level fully typed HTTP client for interacting with Turnkey API
```

Comment:
@r-n-o would this also need to be updated? maybe should be a github permalink or link to the npm registry